src/chronify/time_utils.py

```diff
@@ -70,91 +63,121 @@ def shift_time_interval(
         case _:
             msg = f"Cannot handle from {from_interval_type} to {to_interval_type}"
             raise InvalidParameter(msg)
-    return ser + freq * mult
+    ts_list2 = (arr + freq * mult).tolist()
+    return ts_list2  # type: ignore
 
 
 def wrap_timestamps(
-    ser: "pd.Series[pd.Timestamp]", to_timestamps: list[pd.Timestamp]
-) -> "pd.Series[pd.Timestamp]":
+    ts_list: list[datetime],
+    to_timestamps: list[datetime],
+) -> list[datetime]:
     """Wrap pandas timeseries so it stays within a list of timestamps.
 
     Example:
-    >>> ser = pd.Series(pd.date_range("2018-12-31 22:00", periods=4, freq="h"))
-    0   2018-12-31 22:00:00
-    1   2018-12-31 23:00:00
-    2   2019-01-01 00:00:00
-    3   2019-01-01 01:00:00
-    dtype: datetime64[ns]
+    >>> ts_list = pd.date_range("2018-12-31 22:00", periods=4, freq="h").tolist()
+    [Timestamp('2018-12-31 22:00:00'), Timestamp('2018-12-31 23:00:00'), Timestamp('2019-01-01 00:00:00'), Timestamp('2019-01-01 01:00:00')]
 
     >>> to_timestamps = pd.date_range("2019-01-01 00:00", periods=4, freq="h").tolist()
     [Timestamp('2019-01-01 00:00:00'), Timestamp('2019-01-01 01:00:00'), Timestamp('2019-01-01 02:00:00'), Timestamp('2019-01-01 03:00:00')]
 
-    >>> ser2 = wrap_timestamps(ser, to_timestamps)
-    0   2019-01-01 02:00:00
-    1   2019-01-01 03:00:00
-    2   2019-01-01 00:00:00
-    3   2019-01-01 01:00:00
-    dtype: datetime64[ns]
+    >>> ts_list2 = wrap_timestamps(ts_list, to_timestamps)
+    [Timestamp('2019-01-01 02:00:00'), Timestamp('2019-01-01 03:00:00'), Timestamp('2019-01-01 00:00:00'), Timestamp('2019-01-01 01:00:00')]
```

Comment:
It took me staring at this for a long while for it to make sense. To me, the return value isn't so much "wrapped timestamps" or a "wrapped timeseries" as it is "replacement timestamps in order to wrap data." The name and docstring thus made it harder to understand this example.

Reply:
I'll revise doctoring